quickwit-config/src/config.rs

```diff
@@ -31,7 +31,11 @@ use quickwit_common::uri::{Extension, Uri};
 use serde::{Deserialize, Serialize};
 use tracing::{info, warn};
 
-static DEFAULT_DATA_DIR_PATH: &str = "./qwdata";
+pub const DEFAULT_QW_CONFIG_PATH: &str = "./config/quickwit.yaml";
+
+const DEFAULT_DATA_DIR_PATH: &str = "./qwdata";
+
+const DEFAULT_CLUSTER_ID: &str = "quickwit-test-cluster";
```

Comment:
Nitpik: `quickwit-default-cluster` sounds better to me.

Reply:
Yes, I kept the original name, but I prefer `quickwit-default-cluster` too.